lunavl/sdk/detectors/base.py

```diff
@@ -134,34 +132,43 @@ def getArgsForCoreRedetect(images: List[ImageForRedetection]) -> Tuple[List[Core
     coreImages, detectAreas = [], []
 
     for image in images:
-        assertImageForDetection(image.image)
         coreImages.append(image.image.coreImage)
         detectAreas.append([Detection(bbox.coreRect, 1.0) for bbox in image.bBoxes])
 
     return coreImages, detectAreas
 
 
-def collectAndRaiseError(
-    error: FSDKErrorResult,
-    coreImages: List[CoreImage],
-    detectAreas: Union[List[Detection], List[CoreRectI]],
-    getErrorFunction: Callable[[CoreImage, Union[Detection, CoreRectI]], FSDKErrorResult],
+def validateBatchDetectInput(
+    detector, coreImages: Union[List[CoreImage]], detectAreas: Union[CoreRectI, List[CoreRectI]]
 ) -> None:
     """
     Collect errors from single operations and raise complex exception
     Args:
-        error: fsdk error from core reply
+        detector: core face or body detector
         coreImages: list of core images
         detectAreas: list of detect areas for core images
-        getErrorFunction: function to collect error by
     Raises:
-        LunaSDKException(LunaVLError.BatchedInternalError) with collected errors in context
+        LunaSDKException: if validation are failed or data is not valid
     """
+    if not isinstance(coreImages, list):
+        validationError, imageErrors = detector.validate([coreImages], [detectAreas], 1)
```

Comment:
use keyword for `1`

Reply:
yes... and no=(
![image](https://user-images.githubusercontent.com/5768543/109989367-7d03fd80-7d19-11eb-9032-3a8db0cf7f6c.png)

Reply:
smth like this?
```
limit = 1
validationError, imageErrors = detector.validate([coreImages], [detectAreas], limit)
```

Reply:
+